src/main/io/displayport_msp_bf_compat.c

```diff
@@ -58,42 +58,44 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_DEGREES:
         return BF_SYM_GPS_DEGREE;
 
+/*
     case SYM_HEADING:
-        return BF_SYM_OVER_HOME;
+        return BF_SYM_HEADING;
+
+    case SYM_SCALE:
+        return BF_SYM_SCALE;
 
-/*    case SYM_SCALE:
-        return BF_SYM_SCALE; 
-*/
     case SYM_HDP_L:
-        return 'H';
+        return BF_SYM_HDP_L;
 
     case SYM_HDP_R:
-        return 'P';     // The idea is to form HP, resembling "HDOP"
-
+        return BF_SYM_HDP_R;
+*/
     case SYM_HOME:
         return BF_SYM_HOMEFLAG;
 
     case SYM_2RSS:
         return BF_SYM_RSSI;
 
+/*
     case SYM_DB:
-        return 'D';     // Just D to resemble "dB"
```

Comment:
Not sold on replacing SYM_DB with 'D' and SYM_DBM with blank.

The idea of displaying '?' is to highlight where the problems are. We don't want to hide them.

Reply:
Yeah, replacing dB with just "D" isn't the ideal solution, I agree. But I think it's definitely better than "?". Also, the previous symbol being used for dBm already displayed as blank, I just made it clear. No suitable replacement or letter abbreviation for dBm I could find.

I don't agree at all the "?" should be kept. Highlighting the problems to the developers is one thing, I totally agree with that. Highlighting it for the end user, is pointless. It gives the impression of a buggy and unfinished product. In the initial rush to get the OSD working for the O3 users, it was an acceptable compromise. But I believe enough time has passed for the implementation to become more mature and "polished". Specially when it's just a simple matter of character remapping to greatly improve the user experience.